src/support/hooks.ts

```diff
@@ -148,6 +148,23 @@ Before({tags: '@setup'}, async function(this: ICustomWorld, {pickle}) {
     this.pickle = pickle;
 });
 
+/**
+ * Before each test scenario with the @bwpsetup tag, performs setup tasks.
+ */
+Before({tags: '@bwpsetup'}, async function(this: ICustomWorld, {pickle}) {
+    this.context = await browser.newContext({
+        recordVideo: {
+            dir: "test-results/videos",
+        },
+    });
+    this.page = await this.context.newPage();
+    this.sections = new Sections(this.page, pluginSelectors);
+    this.utils = new PageUtils(this.page, this.sections);
+
+    await this.utils.cleanUp('backwpup');
+    this.pickle = pickle;
+});
```

Comment:
I think we should avoid this kind of change that integrates into wpr test especially when we know we'll be taking it out soon, we can create a new hook script in the backwpup folder and  move this there, then require the hook in the cucumber config.